packages/javascript/js-client-library/src/client.ts

```diff
@@ -304,16 +304,14 @@ class BHEAPIClient {
         dataType: string,
         start?: Date,
         end?: Date,
-        partition_by?: string,
         options?: types.RequestOptions
     ) => {
         return this.baseClient.get<PostureHistoryResponse>(
-            `/api/v2/posture-history/${environmentId}/${dataType}`,
+            `/api/v2/${environmentId}/posture-history/${dataType}`,
```

Comment:
is `/api/v2/domains/{environment_id}/posture-history/{data_type}` according to the spec!